### DISCUSSION TITLE
Lecture Slides

### DISCUSSION THREAD

**Discussion by purnaperamune** [DevinRC/iit-past-assessments#16] [General]
Guys, is it possible any of you to upload lecture slides as well? Or just share a google drive link? (At least for advanced server side module) @DevinRC @Ruchira2k @raminduw200 

Thank you for this repo with past assessments, will be helpful to get ready to students who are going into final year, like me. ❤️

**Comment by DevinRC** [comment]
Hi @purnaperamune, I recommend that you get the lecture slides from the official source, i.e: Blackboard. That way you can make sure you always have the latest and most accurate versions of the slides.

Thanks for your words regarding the repository, it's appreciated 😊

**Reply by purnaperamune** [reply]
Hi @DevinRC, I am starting the final year only in the end of September, just wanted to go through the slides before that. It will be beneficial to me if you can share it. TIA! ❤️
